scripts/ci-e2e.sh

```diff
@@ -115,15 +137,15 @@ if [[ "${IRONIC_BASIC_AUTH}" == "true" ]]; then
   # If usernames and passwords are unset, read them from file or generate them
   if [[ -z "${IRONIC_USERNAME:-}" ]]; then
     if [[ ! -f "${IRONIC_AUTH_DIR}/ironic-username" ]]; then
-      IRONIC_USERNAME="$(uuid-gen)"
+      IRONIC_USERNAME="$(uuidgen)"
```

Comment:
I guess we never have them undefined if `uuid-gen` was ever merged?!

Reply:
I guess so, we (almost) never install without TLS :laughing:

Reply:
That is a good thing on one hand, but OTOH we then have these untested code paths :(